src/main/java/androidx/graphics/Color.kt

```diff
@@ -375,3 +375,8 @@ inline val @receiver:ColorLong Long.isWideGamut get() = Color.isWideGamut(this)
  */
 @get:RequiresApi(26)
 inline val @receiver:ColorLong Long.colorSpace: ColorSpace get() = Color.colorSpace(this)
+
+/**
+ * Return a corresponding [Int] color of this [String].
```

Comment:
This requires more documentation about the valid string formats.

Reply:
Updated